### PR TITLE
Nerfs EMP effect on synthetic limbs/organs

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request

Title.

EMP damage has been reduced significantly, making ion rifles now a 3-4 hit to crit fully augged people.
Paralyze now only happens once the limb is past ~30% damage to maximum, and duration has been significantly reduced.
Knockdown duration on legs reduced.
For the chest, this threshold is 60% and you only get hardstunned on heavy EMP.
EMP ear damage now does much less damage.
## Why It's Good For The Game

EMPs are INSANE against augs right now. 10 second paralyze for a light emp?! TWENTY SECONDS FOR A HEAVY? 10 seconds is enough to win a fight! Even one second is crazy given how it forces a stun/knockdown/forces you to drop your items - hence the damage threshold.

I feel like the current state of EMP balance is entirely based on the nightmare that was flat damage resist augs. As it stands, 20% armor really is not much of a justification to have these absolutely bonkers counters that are so so so easy to get. A single chemist can make a guaranteed kill grenade for a synth in less effort than it takes to make libital!

EMPs still penetrate armor, and ion rifles still crit in 3-4 hits, so it's still really really effective against synths - especially when you consider they still get blinded, fall over, drop their held items, take organ damage...
## Changelog
:cl:
balance: EMP damage on augs: 2/1.5 from 3/2
balance: Augs now only get paralyzed by EMP for 3/6 seconds if they are damaged below 70% HP,
balance: Aug EMP knockdown reduced to 3 seconds
balance: Synthetic ears now take far less EMP damage
/:cl:

### CODE REVIEW

**Comment by nikothedude** [comment]
Atomized from https://github.com/tgstation/tgstation/pull/77920

File: code/modules/surgery/organs/internal/ears/_ears.dm [hunk 1]
```diff
@@ -149,4 +149,4 @@
 	. = ..()
 	if(. & EMP_PROTECT_SELF)
 		return
-	apply_organ_damage(40/severity)
+	apply_organ_damage(10 * severity)
```
Comment (MrMelbert) [inline code/modules/surgery/organs/internal/ears/_ears.dm:152]: But why 

https://github.com/tgstation/tgstation/blob/bbdbe55c69775aa6d558329a309170bc3ee14a1c/code/__DEFINES/combat.dm#L259-L260

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/modules/surgery/organs/internal/ears/_ears.dm (hunk 1 above)
Comment (nikothedude) [reply code/modules/surgery/organs/internal/ears/_ears.dm:152]: holy fucking shit

Comment (nikothedude) [reply code/modules/surgery/organs/internal/ears/_ears.dm:152]: this sucks i hate it why is it like this

**Comment by DaydreamIQ** [comment]
I agree that getting one shot stunned for 10 seconds because some revenant decided to be a funnyman sucks. But I don't like the idea of Androids getting a buff from this considering they're basically the powergamer's choice for wizards

**Comment by CRITAWAKETS** [comment]
finally, i might actually start to see people asking for augs in robotics

**Comment by nikothedude** [comment]
> I agree that getting one shot stunned for 10 seconds because some revenant decided to be a funnyman sucks. But I don't like the idea of Androids getting a buff from this considering they're basically the powergamer's choice for wizards

If a powergamer is picking androids right now they're doing it wrong, androids are fucking terrible from current EMP values

**Comment by SimplySmiley123** [comment]
> I agree that getting one shot stunned for 10 seconds because some revenant decided to be a funnyman sucks. But I don't like the idea of Androids getting a buff from this considering they're basically the powergamer's choice for wizards

Powergamer's choice is the skeleton. Even if healing as a skeleton is harder to find, it's way way less annoying then having to stand still to weld yourself or replaces cables. Plus you don't get stunned by a random person that grinded iron and uranium and mixed both together near you long enough to get murked on the spot.

**Comment by GPeckman** [comment]
> Powergamer's choice is the skeleton. Even if healing as a skeleton is harder to find, it's way way less annoying then having to stand still to weld yourself or replaces cables. Plus you don't get stunned by a random person that grinded iron and uranium and mixed both together near you long enough to get murked on the spot.

IIRC, skeletons can also use normal healing chems in addition to milk. Androids can't use any healing chems at all. Skeletons are definitely the powergamer choice right now.

**Comment by SimplySmiley123** [comment]
> > Powergamer's choice is the skeleton. Even if healing as a skeleton is harder to find, it's way way less annoying then having to stand still to weld yourself or replaces cables. Plus you don't get stunned by a random person that grinded iron and uranium and mixed both together near you long enough to get murked on the spot.
> 
> IIRC, skeletons can also use normal healing chems in addition to milk. Androids can't use any healing chems at all. Skeletons are definitely the powergamer choice right now.

And honestly always was, again, the stun from EMPs was just that ridiculous.

**Comment by nukularpower** [comment]
Completely disagree with this PR - emps SHOULD be the weakness you accept for getting augs, and this goes way too far imo

**Comment by nikothedude** [comment]
> Completely disagree with this PR - emps SHOULD be the weakness you accept for getting augs, and this goes way too far imo

I dont think you understand just how debilitating EMPs are right now. A single EMP completely disables that limb for 10 seconds. A single HEAVY EMP disables it for 20 seconds. A SINGULAR EMP right now is enough to knock out a fully augged person. Its really. REALLY. REALLY bad.

**Comment by ATH1909** [comment]
> > I agree that getting one shot stunned for 10 seconds because some revenant decided to be a funnyman sucks. But I don't like the idea of Androids getting a buff from this considering they're basically the powergamer's choice for wizards
> 
> Powergamer's choice is the skeleton. Even if healing as a skeleton is harder to find, it's way way less annoying then having to stand still to weld yourself or replaces cables. Plus you don't get stunned by a random person that grinded iron and uranium and mixed both together near you long enough to get murked on the spot.

can't you only go skeleton if you buy lichdom? I thought they were specifically excluded from the magic mirror.

did skeletons get the ability to use sutures and healing chems back? if not, then they're still awful as a wiz race because your options for healing yourself as one are kind of exactly "chug milk lmao".

**Comment by SimplySmiley123** [comment]
> Completely disagree with this PR - emps SHOULD be the weakness you accept for getting augs, and this goes way too far imo

On one hand, yes, EMPs should be the nemesis of augments, but at same time 10 to **_20_** seconds of full on stun from something as random as an assistant mixing grinded iron and uranium together?

**Comment by SimplySmiley123** [comment]
> > Completely disagree with this PR - emps SHOULD be the weakness you accept for getting augs, and this goes way too far imo
> 
> I dont think you understand just how debilitating EMPs are right now. A single EMP completely disables that limb for 10 seconds. A single HEAVY EMP disables it for 20 seconds. A SINGULAR EMP right now is enough to knock out a fully augged person. Its really. REALLY. REALLY bad.

Pretty sure yo udon't even need to be fully augmented for the stun to take effect, if I recall just having one body part is enough to get shafted by any EMPs fully.

File: code/modules/surgery/bodyparts/_bodyparts.dm [hunk 2]
```diff
@@ -1292,23 +1291,25 @@
 	. = ..()
 	if(. & EMP_PROTECT_WIRES || !IS_ROBOTIC_LIMB(src))
 		return FALSE
-	owner.visible_message(span_danger("[owner]'s [src.name] seems to malfunction!"))
 
 	// with defines at the time of writing, this is 3 brute and 2 burn
 	// 3 + 2 = 5, with 6 limbs thats 30, on a heavy 60
 	// 60 * 0.8 = 48
```
Comment (Jacquerel) [inline code/modules/surgery/bodyparts/_bodyparts.dm:1297]: This still true? Doesn't look like it is.

**Review by strandsofivy — COMMENTED** [review]
I don't think that using `[src.name]` in strings is necessary, really. Can just use `[src]` afaik

File: code/modules/surgery/bodyparts/_bodyparts.dm [hunk 3]
```diff
@@ -1292,23 +1291,25 @@
...
-	var/time_needed = 10 SECONDS
+	var/time_needed = 0
 	var/brute_damage = AUGGED_LIMB_EMP_BRUTE_DAMAGE
 	var/burn_damage = AUGGED_LIMB_EMP_BURN_DAMAGE
 	if(severity == EMP_HEAVY)
-		time_needed *= 2
+		time_needed = AUGGED_LIMB_EMP_PARALYZE_TIME
```
Comment (Jacquerel) [inline code/modules/surgery/bodyparts/_bodyparts.dm:1302]: This doesn't match your PR body which states that it's 3 seconds light / 6 seconds heavy, only if damaged.
The actual effect seems to be 3 seconds, heavy emp only, only if already damaged.
I like the PR described version better than this one.

**Comment by nikothedude** [comment]
> I don't think that using `[src.name]` in strings is necessary, really. Can just use `[src]` afaik

old code but ill change it in a bit

**Review by Jacquerel — CHANGES_REQUESTED** [review]

File: code/modules/surgery/bodyparts/_bodyparts.dm (hunk 3 above)
Comment (Jacquerel) [reply code/modules/surgery/bodyparts/_bodyparts.dm:1302]: I think losing control of your legs for 3 seconds when emped is not only reasonable but what you would expect to happen if you had robotic legs.

**Comment by nikothedude** [comment]
should mention, didnt document this before, but

chest now only hardstuns you if taken 60%+ damage and on heavy EMP, because FUCK hardstuns

**Pull request merged**